README.md

```diff
@@ -163,6 +163,7 @@ Make sure to choose the appropriate version of this package for the version of X
 
 | Xamarin.Forms  | Xamarin.Forms.Mocks |
 | -------------- | ------------------- |
+| 4.8.0.x        | 4.8.0.x             |
```

Comment:
I usually only update this table if Xamarin.Forms has breaking changes. Did you run into one?

Normally it would be things like this new `GetHash` method that was added to an internal interface:

https://github.com/jonathanpeppers/Xamarin.Forms.Mocks/commit/6079f72a12af9fb19de1a8ae896f8d8cc6263119#diff-7d835bbef44bb212b204866cf0b217eaR55

Reply:
Oh, I see. I didn't find any yet.
Should I just revert the README or do you mean the whole bump isn't needed?

Reply:
It will be fine to merge this. I changed the README, thanks.